automation-test.py

```diff
@@ -0,0 +1,96 @@
+import asyncio
+
+import time
+
+import secterfunction
+
+import automationsectername
+
+from pyppeteer import launch
+
```

Comment:
@sudiptadinda6 Have this inside a class. Call this function from the main.py file. this is the standard approach in python.